### PR TITLE
Closest Distance operation ready

### PR DESCRIPTION
**Pull request by staubibr**
The closest distance operation is ready, the demo_1 workflow has been updated. Also updated tests for spatial_analysis since one operation was trying to run Closest Distance on invalid geometry type. Tests still need to be coded on the closest distance operation, Ghida is looking into that.

For some reason, the model starts generating NaN values after iteration 90 or so. Not sure what that's about but I don't think it's related to the workflow.

### CODE REVIEW

**Review by francoiseblanc — APPROVED** [review]
Looks good @staubibr! Everything works on my side. 

Before merging, you may want to change // for /* as I think we all used this convention. Also, we should be careful not to push "output" folders to ensure that any users running the demos for the first time will run it without error code 50.

I also saw that you changed the name of the workflow file for demo1. We will have to remind ourselves to amend the instructions in the wiki and readme file with the correct filenames :)

**Review by AidanFahlman — APPROVED** [review]
Looks good, great job Bruno!

File: data/demo_1/demo.json [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
-        }, {
-            "id": "vicinities_ottawa",
-            "path": "../data/demo_1/input/DB_Ottawa_Vicinities_Simple.geojson"
+            "id": "DA_Ontario",
+            "path": "../data/demo_1/input/DA_Ontario.geojson"
         }
     ],
     "spatial_analysis": [{
-            "id": "db_centroids",
+            "id": "DA_Ottawa",
```
Comment (romancardenas) [inline data/demo_1/demo.json:9]: Are you overwriting a data source?

File: data/demo_1/demo.json [hunk 2]
```diff
@@ -68,21 +75,21 @@
             "zoom": 5
         },
         "layers": [{
-                "id": "toy-model",
+                "id": "DA",
                 "type": "polygon",
-                "file": "../data/demo_1/input/toy-model.geojson",
-                "label": "Toy Model",
+				"file": "../data/demo_1/input/DA_Ottawa.geojson",
+                "label": "Ottawa DA SIRD",
```
Comment (romancardenas) [inline data/demo_1/demo.json:81]: Should be SIR instead of SIRD

**Review by romancardenas — APPROVED** [review]
Very nice demo example, @staubibr ! Great job.

Additionally, I found the cause of the `nan` issue in the simulation logs. It only happens on cells with a population of 0 people. I don't think we should be worried about that.

**Review by ghidadandan — APPROVED** [review]

**Pull request merged**